app/model/commands/AddAssetCommand.scala

```diff
@@ -63,42 +63,54 @@ case class AddAssetCommand(atomId: String, videoUri: String, override val stores
     UpdateAtomCommand(atomId, fromThrift(updatedAtom), stores, user, awsConfig).process()
   }
 
-  private def getYouTubeChannel(asset: Asset, atom: ThriftMediaAtom): Option[String] = {
+  private def getYouTubeChannel(asset: Asset, atom: ThriftMediaAtom, atomId: String): Option[String] = {
     val existingChannel = atom.metadata.flatMap(_.channelId)
 
-    try {
-      val maybeVideo = youTube.getVideo(asset.id, "snippet")
-
-      (existingChannel, maybeVideo) match {
-        case (_, None) =>
-          Some(YouTubeVideoDoesNotExist(asset.id))
+    def ensureIsGuardianChannel(videoChannel: String) = {
+      if (youTube.channels.exists(_.id == videoChannel)) Some(videoChannel) else IncorrectYouTubeChannel
+    }
 
-        case (None, Some(video)) => {
-          // only GLabs atoms can have third party videos
-          atom.category match {
-            case ThriftCategory.Hosted | ThriftCategory.Paid => Some(video.getSnippet.getChannelId)
-            case _ => {
-              if (youTube.cannotReachYoutube) None
-              else NotGLabsAtom
+    if (youTube.cannotReachYoutube) {
+      log.info(s"Config says cannot reach Youtube")
```

Comment:
what does `Config` mean in this case, does it mean what the user has configured or system config (from S3 etc.)?

Reply:
System configuration - a file on disk which is copied from S3 at instance launch.